docs/plugins/verifier/notation.md

```diff
@@ -260,3 +260,19 @@ Sample Notation CLI config with timestamping configuration:
     }
 }
 ```
+
+## Certificate Revocation Check (CRL)
+
+Ratify supports validating signing identity (certificate and certificate chain) revocation status using [certificate revocation evaluation](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#certificate-revocation-evaluation) as per revocation setting in Notation trust policy. The CRL implementation uses Notation libraries and follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#crls). 
+
+## Intro
+Certificate validation is an essential step during signature validation. Currently Ratify supports checking for revoked certificates through OCSP supported by notation-go library. However, OCSP validation requires internet connection for each validation while CRL could be cached for better performance. As notary-project added the CRL support for notation signature validation, and Ratify utilized it.
+
```

Comment:
This section seems to be just simple introduction but not practical instruction. Shall we consider creating a blog post based on this [CRL bug bash doc](https://hackmd.io/@Juncheng/BJ_lIA7VJg) and link it from this section? Some users might be interested in having a PoC by following a step-by-step guidance before using Ratify.

Reply:
Good point. @FeynmanZhou We can have the post after the bugbash.
Following Susan's comments above I am gonna remove this section for now.